recipe/meta.yaml

```diff
@@ -24,14 +24,16 @@ requirements:
     - m2-patch  # [win]
   host:
     - python
-    - cffi 1.16
+    - cffi 1.16  # [py<313]
+    - cffi 1.17  # [py>312]
```

Comment:
Maybe it's just me, but these pinnings seem weird.

Reply:
I got those from here:  https://github.com/indygreg/python-zstandard/blob/0.23.0/pyproject.toml#L3-L4

Do you have suggestions on how to change it?

Reply:
It would maybe look less weird with `py>=313` instead of `py>312`.

Reply:
```suggestion
    - cffi 1.16  # [py<=312]
    - cffi 1.17  # [py>=313]
```